src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/OrganisationEntityResponse.java

```diff
@@ -91,5 +97,11 @@ private void getOrganisationEntityResponse(
                     .map(ContactInformationResponseWithDxAddress::new)
                     .collect(toList());
         }
+
+        this.dateReceived = organisation.getCreated().toString();
+
+        this.dateApproved = organisation.getDateApproved().toString();

```

Comment:
null pointer exception if date approved is null, make sure that if the date is null then needs to show this json field will null value.

May be check if toString is really required, try to keep fields as LocalDateTime